test/runner.c

```diff
@@ -320,49 +323,59 @@ int run_test(const char* test,
   /* Show error and output from processes if the test failed. */
   if (status != 0 || task->show_output) {
     if (tap_output) {
-      LOGF("#");
+      fprintf(stderr, "#");
+      fflush(stderr);
     } else if (status == TEST_TODO) {
-      LOGF("\n`%s` todo\n", test);
+      fprintf(stderr, "\n`%s` todo\n", test);
+      fflush(stderr);
     } else if (status == TEST_SKIP) {
-      LOGF("\n`%s` skipped\n", test);
+      fprintf(stderr, "\n`%s` skipped\n", test);
+      fflush(stderr);
     } else if (status != 0) {
-      LOGF("\n`%s` failed: %s\n", test, errmsg);
+      fprintf(stderr, "\n`%s` failed: %s\n", test, errmsg);
+      fflush(stderr);
     } else {
```

Comment:
Instead of one inside each, why not just have one `fflush(stderr)` after the if statements?